bot/connector-iadvize/src/test/kotlin/IadvizeConnectorMessageTest.kt

```diff
@@ -0,0 +1,112 @@
+package ai.tock.bot.connector.iadvize/*
```

Comment:
package en haut du copyright
```suggestion
/*
```